src/components/PostsList/PostsList.tsx

```diff
@@ -1,37 +1,83 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './PostsList.scss';
+import { getAllPosts, getPosts } from '../../api/posts';
+import { Post } from '../../react-app-env';
```

Comment:
It's a good practice to order and group the imports. You can group external imports first, followed by internal imports. Also, ordering them alphabetically can make it easier to manage.